doc/modules/linear_model.rst

```diff
@@ -71,6 +77,16 @@ this method has a cost of
 :math:`O(n_{\text{samples}} n_{\text{features}}^2)`, assuming that
 :math:`n_{\text{samples}} \geq n_{\text{features}}`.
 
+Cholesky Complexity
+---------------------------------
+
+The Cholesky solution is computed using the Cholesky factorization of
+X. If X is a matrix of shape `(n_samples, n_features)` this  method has
```

Comment:
The cholesky solver solves the normal equation by a Cholesky decomposition of X'X or XX', whichever has smaller dimension. That's why the condition number of of problem is doubled and the numerical solution can become more unstable compared to approaches that use decompositions of X alone like lstsq does.

Reply:
I think this would be interesting to expand the doc to include this remark, possibly as a new paragraph.